R/model_text_parameters.R

```diff
@@ -235,7 +235,7 @@ model_text_parameters_bayesian <- function(model, parameters, ci = 0.90, rope_fu
   }
 
   if (all(estimate_full != "")) {
-    estimate <- paste0(string_split_matrix(estimate_full, ", ")[, 1], ", ")
+    estimate <- paste0(string_split_matrix(estimate_full, ", ")[1, ], ", ")
```

Comment:
This function was a bit hairy, but I hope it works as intended...

Reply:
fixed 🤞